### ISSUE TITLE
overriding default login

### ISSUE THREAD

**Issue by arist0v** [NginxProxyManager/nginx-proxy-manager#3805]
**Is your feature request related to a problem? Please describe.**

I would like to be able to set via environnement variable the default login so i could deploy my image with my own login at start.

**Describe the solution you'd like**

ENV DEFAULT_USER
ENV DEFAULT_PASSWORD
adding those 2 env variable could help people override the settings

**Describe alternatives you've considered**

**Additional context**

**Comment by alex190291** [comment]
yeah i would love this too. it would really help me streamline the setup of the all-in-one webhosting panel im working on right now that uses npm as its reverse proxy. i implemented my own frontend for npm (mainly because of the missing dark mode and too many sub menus) using the swagger-api/OpenAPI interface but the lack of documentation made this a royal pain in the arse so it would be nice if i wouldnt need to implement a "simulated" first login just to change the email/password to a value the user provides in my apps config.yml.
